### ISSUE TITLE
False positive for `no-commented-out-code` with *hyphen* and *slash*

### ISSUE THREAD

**Issue by Arttse** [cartant/eslint-plugin-etc#33]
For example: 
```ts
enum MyExample {
  // all-powerful
  Zero,
  /**
   * play-group
   */
  One,
  // and/or
  Two,
  /**
   * his/her
   */
  Three,
}
```

---

Plugin version `1.5.3`

**Comment by cartant** [comment]
Should be fixed in 1.5.4.
